sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -448,4 +456,120 @@ export class ApplicationStudentsController extends BaseController {
       applicationNumber: application.applicationNumber,
     };
   }
+
+  /**
+   * Get in progress details of an application by application id.
+   * @param applicationId application id.
+   * @returns application details.
+   */
+  @Get(":id/in-progress")
+  @ApiNotFoundResponse({
+    description: "Application id not found.",
+  })
+  async getInProgressApplicationDetails(
+    @Param("id", ParseIntPipe) applicationId: number,
+  ): Promise<InProgressApplicationDetailsAPIOutDTO> {
+    const application = await this.applicationService.getApplicationByIdAndUser(
+      applicationId,
+    );
+    if (!application) {
+      throw new NotFoundException(
+        `Application id ${applicationId} was not found.`,
+      );
+    }
+
+    const incomeVerificationDetails =
+      await this.craIncomeVerificationService.allIncomeVerificationsForAnApplication(
+        applicationId,
+      );
+    const incomeVerification =
+      this.applicationControllerService.processApplicationIncomeVerificationDetails(
+        incomeVerificationDetails,
+      );
+
+    const supportingUserDetails =
+      await this.supportingUserService.getSupportingUsersByApplicationId(
```

Comment:
Please keep it consistent between `process` or `get` as prefixes.

Reply:
so, I used `get` to get data from DB and `process` to transform the data to a dto